tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -420,12 +433,13 @@ void testing_spi(void)
 	spi_slow = device_get_binding(SPI_DRV_NAME);
 	if (!spi_slow) {
 		SYS_LOG_ERR("Cannot find %s!\n", SPI_DRV_NAME);
+		zassert_not_null(spi_slow, "Invalid SPI device");
```

Comment:
We enter the `if` block if it's null, so can we assert false directly instead of recheck, like you've done for some others?

Reply:
The recheck, while technically redundant, makes the intent of the assertion more clear. I considered removing the SYS_LOG_ERR's altogether and using zasserts exclusively, but that had a much greater impact on the flow of the testcase. I asserted false directly only after memcmp checks.

Reply:
Okay, makes sense, thanks